cranelift/filetests/filetests/isa/x86/simd-arithmetic-run.clif

```diff
@@ -192,31 +192,22 @@ block0:
 }
 ; run
 
-function %fmax_f64x2() -> b1 {
-block0:
-    v0 = vconst.f64x2 [-0.0 -0x1.0]
-    v1 = vconst.f64x2 [+0.0 +0x1.0]
-
+function %fmax_f64x2(f64x2, f64x2) -> f64x2 {
+block0(v0: f64x2, v1: f64x2):
     v2 = fmax v0, v1
-    v3 = fcmp eq v2, v1
-    v4 = vall_true v3
-
-    return v4
+    return v2
 }
-; run
-
-function %fmin_f64x2() -> b1 {
-block0:
-    v0 = vconst.f64x2 [-0x1.0 -0x1.0]
-    v1 = vconst.f64x2 [+0.0 +0x1.0]
+; run: %fmax_f64x2([-0x0.0 -0x1.0], [+0x0.0 0x1.0]) == [+0x0.0 0x1.0]
+; run: %fmax_f64x2([-NaN NaN], [0x0.0 0x100.0]) == [-NaN NaN]
 
+function %fmin_f64x2(f64x2, f64x2) -> f64x2 {
+block0(v0: f64x2, v1: f64x2):
     v2 = fmin v0, v1
-    v3 = fcmp eq v2, v0
-    v4 = vall_true v3
-
-    return v4
+    return v2
 }
-; run
+; run: %fmin_f64x2([-0x0.0 -0x1.0], [+0x0.0 0x1.0]) == [-0x0.0 -0x1.0]
+; run: %fmin_f64x2([-NaN 0x100.0], [0x0.0 NaN]) == [-NaN -NaN]  ; note how the second NaN is now quieted and negative:
+; this is due to non-determinism in the NaN sequence.
```

Comment:
Can you point in the wasm SIMD spec document (in a Github comment) where it is stated that it's to lose the NaN's sign, please? I couldn't find anything in the proposal design doc.

Reply:
Yeah, I added the link here and in `enc_tables.rs` but here it is as well: https://webassembly.github.io/spec/core/bikeshed/index.html#nan-propagation%E2%91%A0